vm.go

```diff
@@ -337,8 +337,14 @@ func (vm *vm) relocateHyperCommand(hyper *api.Hyper) error {
 
 	for _, cmd := range cmds {
 		if hyper.HyperName == cmd.name {
-			if nTokens != 1 {
-				return fmt.Errorf("expected 1 token, got %d", nTokens)
+			if nTokens > 1 {
+				return fmt.Errorf("expected 0 or 1 token, got %d", nTokens)
+			}
+
+			// We don't need a token to start a process inside the VM, but we
+			// don't get a session in that case.
+			if nTokens == 0 {
```

Comment:
Not really, it's business as usual.

It may be worth sprinkling more info messages in general, that's something I'll have a look at when improving the logging story in 3.0 (see https://github.com/clearcontainers/proxy/issues/35, https://github.com/clearcontainers/proxy/issues/35)